package.json

```diff
@@ -126,7 +125,7 @@
     "nock": "^9.0.22",
     "nsp": "^3.0.0",
     "nyc": "^12.0.2",
-    "pa11y": "^4.6.0",
+    "pa11y": "^5.0.0",
```

Comment:
shouldn't this be 5.0.4?

Reply:
Could be, but it does not really matter because yarn.lock pins to 5.0.4 (latest version in range >=5.0.0 and <6.0.0.